components/Lists/list.css

```diff
@@ -0,0 +1,35 @@
+.list-style-none{
+    list-style: none;
+    padding-inline-start: 0;
+}
+.list-pd{
+    border: 1px solid white;
+    padding: 1rem 5rem;
+}
+.styled-list{
+    margin-bottom: 1rem;
+}
+.bg{
```

Comment:
please give the proper classname. The name of a class is bg but it also includes height and width.

Reply:
ok done